docs/pages/database-access/guides/azure-redis.mdx

```diff
@@ -27,17 +29,13 @@ This guide will help you to:
 - `redis-cli` version `6.2` or newer installed and added to your system's
   `PATH` environment variable.
 
-(!docs/pages/includes/user-client-prereqs.mdx!)
-
-## Step 1/6. Install Teleport
-
-(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+(!docs/pages/includes/tctl.mdx!)
 
-## Step 2/6. Create a Teleport user
+## Step 1/5. Create a Teleport user
```

Comment:
Should a user be added to the prerequisites for these guides, or do we specifically want a new Teleport user just for database access?

Reply:
I wanted to reduce the scope of this PR, so I made minimal changes aside from removing the `start-auth-proxy.mdx` instructions. So far, our guides are split between creating a user and adding a role to a user.